packages/@aws-cdk/aws-apprunner-alpha/README.md

```diff
@@ -215,3 +215,24 @@ const service = new apprunner.Service(stack, 'Service', {
 
 service.addSecret('LATER_SECRET', apprunner.Secret.fromSecretsManager(secret, 'field'));
 ```
+
+## HealthCheck Configuration
+
+To configure the health check for the service, use the `healthCheckConfiguration` attribute.
+
+```ts
+new apprunner.Service(this, 'Service', {
+  source: apprunner.Source.fromEcrPublic({
+    imageConfiguration: { port: 8000 },
+    imageIdentifier: 'public.ecr.aws/aws-containers/hello-app-runner:latest',
+  }),
+  healthCheckConfiguration: {
+    healthyThreshold: 5,
+    interval: Duration.seconds(10),
+    path: '/',
+    protocol: apprunner.HealthCheckProtocolType.HTTP,
+    timeout: Duration.seconds(10),
+    unhealthyThreshold: 10,
+  },
+});
```

Comment:
I'd like us to use an enum like class for this, because HTTP and TCP health checks have different props:
```suggestion
new apprunner.Service(this, 'Service', {
  source: apprunner.Source.fromEcrPublic({
    imageConfiguration: { port: 8000 },
    imageIdentifier: 'public.ecr.aws/aws-containers/hello-app-runner:latest',
  }),
  healthCheck: apprunner.HealthCheck.http({
    healthyThreshold: 5,
    interval: Duration.seconds(10),
    path: '/',
    timeout: Duration.seconds(10),
    unhealthyThreshold: 10,
  }),
});
```

Have a look at `appmesh.HealthCheck`

Reply:
@mrgrain 

I changed to use the `HealthCheck` class like `appmesh.HealthCheck`.

https://github.com/aws/aws-cdk/pull/27029/commits/d0cd25506a66ab4c62547827fbe7410a40891263